backend/functions/lib/adapters/types/basic.d.ts

```diff
@@ -9,9 +10,19 @@ export interface AppConfig {
   server: ServerType;
 }
 
+export interface LoggerHelper {
+  debug: (moduleName: string, message: string, data?: unknown) => void;
+  info: (moduleName: string, message: string, data?: unknown) => void;
+  warn: (moduleName: string, message: string, data?: unknown) => void;
+  error: (moduleName: string, message: string, data?: unknown) => void;
+  camsError(erro: CamsError);
+}
+
+// TODO: By extending Context from Azure Functions we have allowed Azure to invade this codebase.
```

Comment:
We've wrapped the Azure `Context` object. What other way would we want to limit our exposure to risk from this third-party component?

I suppose we could write an interface and an adapter, but that adapter is still going to either wrap the Azure `Context` or we'll have to destructure it.